### PR TITLE
2237 rust side

### LINKED ISSUE #2237: Add rust2eo communication to rust object
At the moment there is no any way to manage eo object from `rust` object. 
We need to change its code signature to `fn foo(mut env: EOenv) -> Result<i32>`. Where `EOenv` is a structure that provides managing eo objects. `EOenv`  is an analogue  of [`JNIEnv`](https://docs.oracle.com/javase/8/docs/technotes/guides/jni/spec/functions.html#interface_function_table) and needs to have  the methods below:

1. `find(&self, obj: &str) -> Option::<u32>` Find and returns the vertex of the eo object. obj is the path to the object, for instance, `"^.^.other-obj"`. #2267
2. `put(&self, v: u32, bytes: &[u8])` . `bytes` can be for example `&[0x00, 0x1a, 0xEE, 0xf, 0xf3]`.
3. `bind(&self, v1: u32, v2: u32, name: &str) -> Option<()>`  Binds v2 to v1. Thus v1 has attribute `name` with vertex = v2.
4. `copy(&self, v: u32) -> Option<u32>` Copies the object and returns the vertex of new object.
5. ` dataize(&self, v: u32) -> Option<&[u8]>` Dataizes the object and returns data's bytes.
@yegor256 WDYT?

### PR DESCRIPTION
**Pull request by levBagryansky**
Closes #2237

---

### Focus of this PR:
This PR focuses on implementing new methods in the `EOEnv` Rust struct and the corresponding Java class `EOrust` to enable data manipulation in EO objects.

### Detailed summary:
- Implemented the `put` method in `EOEnv` to store data in EO objects.
- Implemented the `bind` method in `EOEnv` to establish a parent-child relationship between EO objects.
- Implemented the `copy` method in `EOEnv` to create a copy of an EO object.
- Added placeholder code for the `dataize` method in `EOEnv` to convert EO object data into a byte array.
- Added placeholder code for the `put`, `bind`, and `copy` methods in the `EOrust` Java class.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by levBagryansky** [comment]
@Graur please review

**Review by Graur — CHANGES_REQUESTED** [review]

File: eo-runtime/src/test/eo/org/eolang/rust-tests.eo [hunk 1]
```diff
@@ -49,11 +49,82 @@
...
+    """
+    *
+  assert-that > @
+    copy
+    $.not
+      $.less-than
+        0
+
+[] > rust-dataize-not-fails
+  nop > @
```
[the remark this answers is not in the document]
Comment (levBagryansky) [reply eo-runtime/src/test/eo/org/eolang/rust-tests.eo:115]: @Graur Indeed we can enable it

**Review by Graur — APPROVED** [review]
@levBagryansky Looks good to me, thanks!
@yegor256 Please have a look

**Comment by levBagryansky** [comment]
@yegor256 could you please merge these changes?

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**